crates/task-impls/src/view_sync.rs

```diff
@@ -743,6 +744,12 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                         .publish(HotShotEvent::ViewSyncFinalizeVoteSend(vote))
                         .await;
                 }
+
+                error!(
+                    "View sync protocol has received view sync evidence to update the view to {}",
+                    *self.next_view
+                );
```

Comment:
I think the logging at lines 475 and 484 should suffice?

Reply:
My thought here was to log when view sync successfully updates the view, as well as when it starts.  So we can see if view sync is updating the view or if the view is updated because a proposal was received.

Reply:
Makes sense!